### PR TITLE
Dependencies and Additional tests

### PR DESCRIPTION
**Pull request by 7thcubic**
Fixed dependencies issues for testing.
Changes to chr line feed to cross-platform php constant
Added test for add spread compiler

Overall low priority to merge in.

### CODE REVIEW

**Comment by 7thcubic** [comment]
Also you might want to remove 5.3 from travis. Cause support from the trusty image doesn't do php 5.3, 5.6 might be a good thing, since 5.6 is the final version of php5, while adding support for php 7+ on travis might be a good idea

File: Tests/Units/DependencyInjection/Compiler/AddSpreadCompilerPass.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Spy\TimelineBundle\Tests\Units\DependencyInjection\Compiler;
+
+use mageekguy\atoum;
+use Spy\TimelineBundle\DependencyInjection\Compiler\AddSpreadCompilerPass as TestedModel;
+
+class AddSpreadCompilerPass extends atoum\test
```
Comment (core23) [inline Tests/Units/DependencyInjection/Compiler/AddSpreadCompilerPass.php]: Test suffix is missing

Comment (7thcubic) [reply Tests/Units/DependencyInjection/Compiler/AddSpreadCompilerPass.php]: actually i followed the convention of AddLocatorCompilerPass

**Comment by stephpy** [comment]
Indeed, i'll remove php 5.3 tests in travis. Thanks.

**Comment by stephpy** [comment]
Tests are now green 🍻

**Comment by stephpy** [comment]
I'll take some time soon to implement php 7.0/7.1

**Pull request merged**